package.json

```diff
@@ -59,10 +59,10 @@
     "typescript-tslint-plugin": "0.3.1"
   },
   "peerDependencies": {
-    "@angular-devkit/architect": ">=0.7.2",
-    "@angular-devkit/build-angular": ">=0.7.2",
-    "@angular-devkit/core": ">=0.7.2 <8.0.0",
-    "@angular-devkit/schematics": ">=0.7.2 <8.0.0"
+    "@angular-devkit/architect": "0.800.0",
```

Comment:
Could you change this and the following lines to something like "@angular-devkit/architect": ">=0.800.0",

Otherwise this happens: `@angular-devkit/architect@0.800.0 but none is installed. You must install peer dependencies yourself.` if you have a version newer than 0.800.0 (like me because I have 0.800.1)

The same for the following lines

Reply:
Agreed. Updated. cc @tlancina